tests/test_smoke.py

```diff
@@ -49,6 +49,8 @@
 from sky.utils import common_utils
 from sky.utils import subprocess_utils
 from sky.clouds import AWS, GCP, Azure
+from sky.adaptors import ibm
+from sky.data.data_utils import Rclone
```

Comment:
nit: alphabetical ordering for imports

Reply:
Done. Hope I understood you, since i had to reorder 2 other import lines in the process.